crashbin_app/templates/crashbin_app/report_detail.html

```diff
@@ -75,7 +75,7 @@ <h1 class="m-0 py-2">Report Details</h1>
                                                 {% endif %}
                                                 " role="alert">
                                         <p>{{ message.NAME }} from <strong>{{ message.author_str }}</strong><br>at {{ message.created_at.ctime }}:</p>
-                                        {{ message.contents | linebreaks }}
+                                        <p class="m-0 text-wrap">{{ message.contents }}</p>
```

Comment:
I don't think this is a good idea. If someone writes:

```
- foo
- bar
- baz
```

that should show up like this, not as `- foo - bar - baz`.